### PR TITLE
Update entrepreneurship-training.md

### PR DESCRIPTION
**Pull request by lanaduffy**
Updated copy

### CODE REVIEW

**Comment by emilyville** [comment]
not sure what's happening here, but it's complaining about this link:

- ./_site/education/advanced-training-and-certifications/entrepreneurship-training/index.html
  *  internally linking to /education/legacy-programs/veap, which does not exist (line 175)

Which seems to load for me. @awong-dev ?

**Comment by awong-dev** [comment]
Huh. @ayaleloehr: we need a way to tie the reverse-proxy configs to the htmlproof exclude checks to fix this systemtically.  Would love ideas.

For now though, is there a place where I can find all those configs so I can update htmlproof's commandline in one go?

**Comment by ayaleloehr** [comment]
@awong-dev this link is a ` /education` link, not `/employment`, so it shouldn't have anything to do with the reverse proxy.

**Comment by ayaleloehr** [comment]
Here's what I'm seeing: when I build locally, I do not get a folder called `./_site/education/legacy-programs`.  Looking at the production S3 bucket, this folder *does* exist there, however. 

@awong-dev Do the S3 buckets get deleted each time they are rewritten to?  I.e. if a file or folder gets deleted when a build happens, will it be deleted in staging and production S3 buckets after the next merge?  If no, let's move this conversation to another issue to figure out the consequences of that. 

@emilyville I think the correct path is `/education/other-educational-assistance-programs/veap/`.  Assigning to you to decide if that's correct and make the change.

**Comment by awong-dev** [comment]
@ayaleloehr hmm...okay, I think I need a primer on the nginx config at some point (or at least a pointer to the repo with the configs) but agreed on your analysis. We do NOT clear the bucket before redeploying. Let's move to another issue.

**Comment by awong-dev** [comment]
In fact, let's move it to #957

**Comment by emilyville** [comment]
Assigning to you @awong-dev (or @ayaleloehr ) since I am sure what the next steps are for this.

File: _education/advanced-training-and-certifications/entrepreneurship-training.md [hunk 1]
```diff
@@ -20,8 +20,23 @@ template: 6-info-page
...
-Learn more about [entrepreneurship training](http://www.benefits.va.gov/GIBILL/docs/factsheets/Entrepreneurship_Training.pdf).
+You do not have to use GI Bill funding to participate in this program. 
 
+### Are you eligible for this training?
+
+Yes, if you are eligible for any of the following:
+
+- Montgomery GI Bill, [Active Duty](/education/gi-bill/montgomery-active-duty) or [Selected Reserve](/education/gi-bill/montgomery-selected-reserve)
+- [Post-9/11 GI Bill](/education/gi-bill/post-9-11)
+- [VEAP](/education/legacy-programs/veap)
```
Comment (ayaleloehr) [inline _education/advanced-training-and-certifications/entrepreneurship-training.md]: @emilyville No technical fix needed here, it's simply a wrong link. VEAP is no longer at `/education/legacy-programs/veap` but is at `/education/other-educational-assistance-programs/veap/`.  So changing this line should make the PR green.

**Comment by ayaleloehr** [comment]
@emilyville added a comment on how to make the PR green, but I want to make sure you see this and it makes sense to you before it gets merged (mostly so you're aware the VEAP page moved from `/legacy-programs` to `/other-educational-assistance-programs`).

**Comment by emilyville** [comment]
K. I will assign to @kantrowitz to fix. Can you fix these tests by updating the link as outlined above? Let me know if you need help.

**Comment by emilyville** [comment]
And thanks, @awong-dev and @ayaleloehr !

**Comment by emilyville** [comment]
LGTM! :+1:

**Pull request merged**